x-pack/plugins/uptime/public/lib/alert_types/monitor_status.tsx

```diff
@@ -44,15 +46,27 @@ const AlertMonitorStatus = React.lazy(() =>
 );
 
 export const initMonitorStatusAlertType: AlertTypeInitializer = ({
-  autocomplete,
+  store,
+  core,
+  plugins,
 }): AlertTypeModel => ({
   id: CLIENT_ALERT_TYPES.MONITOR_STATUS,
-  name: <MonitorStatusTitle />,
-  iconClass: 'uptimeApp',
-  alertParamsExpression: (params: any) => (
-    <AlertMonitorStatus {...params} autocomplete={autocomplete} />
+  name: (
+    <ReduxProvider store={store}>
+      <MonitorStatusTitle />
+    </ReduxProvider>
   ),
+  iconClass: 'uptimeApp',
+  alertParamsExpression: (params: any) => {
+    return (
+      <ReduxProvider store={store}>
+        <KibanaContextProvider services={{ ...core, ...plugins }}>
+          <AlertMonitorStatus {...params} autocomplete={plugins.data.autocomplete} />
+        </KibanaContextProvider>
+      </ReduxProvider>
```

Comment:
For store changes, we only need this part, it's not necessary to initiate it in plugin start. You just need to import store from it's current place and add this part. Essentially it just need a redux provider.

Reply:
Right, we discussed this previously. I'll add a commit today to verify this and revert to the current method of the store's initialization.